src/components/molecules/MoleculeSRPCard.vue

```diff
@@ -76,121 +91,138 @@ export default {
 .custom-card {
+    margin: 0.8rem;
     border-radius: var(--border-default);
     box-shadow: 0 0.5em 1em -0.125em rgb(10, 10, 10, 10%),
         0 0 0 1px rgb(10, 10, 10, 20%);
-    column-gap: 0.25rem;
-    display: grid;
-    grid-template-columns: repeat(6, 1fr);
-    grid-template-rows: repeat(5, 1fr);
-    height: 220px;
-    margin: 0.5rem;
-    max-width: 500px;
     overflow: hidden;
+    max-width: 500px;
     padding: 1rem;
-    border: 2px solid transparent;
-    transition: transform 0.3s ease;
-}
-
-.custom-card:hover {
-    border: 2px solid var(--secondary-color);
-    transform: scale(1.03);
-}
-.card-title {
-    font-size: 1rem;
-    grid-column: 1 / 5;
-    grid-row: 1 / 1;
-    -webkit-box-orient: vertical;
-    display: -webkit-box;
-    -webkit-line-clamp: 1;
-    overflow: hidden;
-}
-
-.card-location {
-    align-items: center;
-    display: flex;
-    font-size: var(--sp-size-sm);
-    gap: 10px;
-    grid-column: 1 / 5;
-    grid-row: 2 / 2;
     position: relative;
-    line-height: 1.1;
-    color: #6b6b6b;
-}
-
-.location-icon {
-    font-size: 20px;
-}
-
-.location-text {
-    -webkit-box-orient: vertical;
-    display: -webkit-box;
-    -webkit-line-clamp: 2;
-    overflow: hidden;
-}
-
-.card-rating {
-    grid-column: 1 / 3;
-    grid-row: 3 / 3;
-    margin-top: 0.75rem;
-}
-
-.card-distance {
-    background-color: rgb(214, 253, 255);
-    width: 58px;
-    text-align: center;
-    padding: 0.75rem 0.75rem;
-    border-radius: var(--border-default);
-    grid-column: 5 / 7;
-    grid-row: 1 / 3;
-    align-self: end;
-    justify-self: end;
-}
-
-.distance-text {
-    line-height: var(--lh-small);
-    font-weight: var(--bold-font);
-    font-size: 16px;
-}
-
-.text-color {
-    color: var(--secondary-color);
-}
-
-.card-rate {
-    align-items: center;
-    display: flex;
-    font-weight: var(--bold-font);
-    gap: 10px;
-    grid-column: 1 / 4;
-    grid-row: 4 / 5;
-    margin-bottom: 0.75rem;
-}
-
-.rate-icon {
-    color: green;
-}
-
-.card-spot {
-    grid-column: 4 / 6;
-    grid-row: 4 / 5;
-}
-
-.card-spot-red {
-    color: red;
-    font-weight: var(--bold-font);
-}
-
-.card-spot-green {
-    color: green;
-    font-weight: var(--bold-font);
-}
-
-.card-btn {
-    grid-column: 1 / 7;
-    grid-row: 5 / 6;
-    font-weight: var(--bold-font);
+    transition: transform 0.3s ease;
+    cursor: pointer;
+
+    &:hover {
+        transform: scale(1.03);
+
+        &::before {
+            content: '';
+            position: absolute;
+            background: var(--secondary-color);
+            bottom: 25%;
+            left: 0;
+            width: 2px;
+            height: 50%;
+        }
+
+        &::after {
+            content: '';
+            position: absolute;
+            background: var(--secondary-color);
+            right: 0;
+            bottom: 25%;
+            width: 2px;
+            height: 50%;
+        }
+    }
+
+    .card-primary {
+        display: flex;
```

Comment:
nit: keep attributes alphabetically sorted.

Reply:
not following the alphabetic order..
below is the order of attributes
            'position',
            'top',
            'right',
            'bottom',
            'left',
            'z-index',
            'display',
            'justify-content',
            'align-items',
            'float',
            'clear',
            'overflow',
            'overflow-x',
            'overflow-y',
            'margin',
            'margin-top',
            'margin-right',
            'margin-bottom',
            'margin-left',
            'padding',
            'padding-top',
            'padding-right',
            'padding-bottom',
            'padding-left',
            'width',
            'min-width',
            'max-width',
            'height',
            'min-height',
            'max-height',
            'font-size',
            'font-family',
            'font-weight',
            'border',
            'border-style',
            'border-width',
            'border-color',
            'border-top',
            'border-top-style',
            'border-top-width',
            'border-top-color',
            'border-right',
            'border-right-style',
            'border-right-width',
            'border-right-color',
            'border-bottom',
            'border-bottom-style',
            'border-bottom-width',
            'border-bottom-color',
            'border-left',
            'border-left-style',
            'border-left-width',
            'border-left-color',
            'border-radius',
            'text-align',
            'text-justify',
            'text-indent',
            'text-overflow',
            'text-decoration',
            'white-space',
            'color',
            'background',
            'background-position',
            'background-repeat',
            'background-size',
            'background-color',
            'background-clip',
            'opacity',
            'filter',
            'list-style',
            'outline',
            'visibility',
            'box-shadow',
            'text-shadow',
            'resize',
            'transition',

Reply:
for where you got the list?